src/tests/jsx.rs

```diff
@@ -143,6 +143,49 @@ to!(
     "#
 );
 
+to!(
+    jsx_explicit_labels,
+     r#"
+       import { Trans } from "@lingui/react/macro";
+
+       <Trans>Refresh {{foo}} inbox</Trans>;
+       <Trans>Refresh {{foo: foo.bar}} inbox</Trans>;
+       <Trans>Refresh {{foo: expr()}} inbox</Trans>;
+       <Trans>Refresh {{foo: bar, baz: qux}} inbox</Trans>;
+       <Trans>Refresh {{}} inbox</Trans>;
+       <Trans>Refresh {{...spread}} inbox</Trans>;
```

Comment:
I think both of them could be left as-is, but ensure that the eslint rule is correctly flagging these cases with a proper error message (Invalid placeholder bla bla)